composer/trainer/trainer.py

```diff
@@ -946,6 +974,17 @@ def __init__(
                     stream=console_stream,
                 ))
 
+        if checkpoint_save_path is not None:
```

Comment:
I think `checkpoint_save_path` is undefined?

Reply:
fixed?